donkeycar/parts/lidar.py

```diff
@@ -737,6 +737,11 @@ def run(self, measurements):
         plot_polar_angle(draw, bounds, self.border_color, 0,
                          self.angle_direction, self.rotate_plot)
         
+        # handle no data yet
+        if measurements is None:
```

Comment:
maybe this can be 
```python
# data points
if measurements:
  plot_polar_points(
      draw, bounds, self.mark_fn, self.point_color, self.mark_px,
      [(distance, angle) for distance, angle, _, _, _ in measurements],
      self.max_distance, self.angle_direction, self.rotate_plot)

```